Makefile.linux.mk

```diff
@@ -87,7 +77,6 @@ clean-submodules:
 	cd frida-python && git clean -xfd
 	cd frida-node && git clean -xfd
 
-
```

Comment:
The two newlines are intentional and used to visually separate distinct sections.